pyvista/core/pointset.py

```diff
@@ -360,7 +362,6 @@ def save(self, filename, binary=True):
         -----
         Binary files write much faster than ASCII and have a smaller
         file size.
```

Comment:
```suggestion
         file size.

```
PyCodestyle is upset about this